### PR TITLE
Feature/selc 175

### PR DESCRIPTION
**Pull request by rodoviti**

### CODE REVIEW

File: core/src/test/java/it/pagopa/selfcare/dashboard/core/PartyManagementServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    private PartyManagementRestClient restClientMock;
+
+
+    @Test
+    void getOrganization() {
+        // when
+        partyManagementService.getOrganization(null);
+        // then
+        // verifica che il metodo getOrganization del restClientMock sia stato invocato una volta
+        Mockito.verify(restClientMock, Mockito.times(1)).getOrganization(null);
```
Comment (antonioT90) [inline core/src/test/java/it/pagopa/selfcare/dashboard/core/PartyManagementServiceImplTest.java:27]: Consider to verify no other interaction with the mock:
https://www.javadoc.io/doc/org.mockito/mockito-core/2.7.9/org/mockito/Mockito.html#verifyNoMoreInteractions(java.lang.Object...)

File: web/src/main/java/it/pagopa/selfcare/dashboard/web/config/DashboardSecurityConfig.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    @Autowired
+    public DashboardSecurityConfig(JwtService jwtService, PartyProcessRestClient restClient) {
+        super(jwtService);
+        this.restClient = restClient;
+    }
+
+
+    @Override
+    public void configure(AuthenticationManagerBuilder authenticationManagerBuilder) {
```
Comment (antonioT90) [inline web/src/main/java/it/pagopa/selfcare/dashboard/web/config/DashboardSecurityConfig.java:30]: Should this configuration be implemented in SecurityConfig?

File: integration/rest/src/test/java/it/pagopa/selfcare/dashboard/connector/rest/client/PartyManagementRestClientTest.java [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                .extensions(new ResponseTemplateTransformer(false));
+        config.mappingSource(new JsonFileMappingsSource(config.filesRoot().child("stubs/party-mgmt")));
+        wireMockRule = new WireMockClassRule(config);
+    }
+
+
+    public static class RandomPortInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+        @SneakyThrows
+        @Override
+        public void initialize(ConfigurableApplicationContext applicationContext) {
```
Comment (antonioT90) [inline integration/rest/src/test/java/it/pagopa/selfcare/dashboard/connector/rest/client/PartyManagementRestClientTest.java:55]: common logic behavior as precedent comment?

File: integration/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/config/PartyManagementRestClientConfig.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@Import(RestClientBaseConfig.class)
+@EnableFeignClients(clients = PartyManagementRestClient.class)
+@PropertySource("classpath:config/party-mgmt-rest-client.properties")
+class PartyManagementRestClientConfig {
```
Comment (antonioT90) [inline integration/rest/src/main/java/it/pagopa/selfcare/dashboard/connector/rest/config/PartyManagementRestClientConfig.java:15]: How are you handling the Authentication header forward to invoke REST secured api?

**Review by antonioT90 — CHANGES_REQUESTED** [review]

File: web/src/main/java/it/pagopa/selfcare/dashboard/web/config/DashboardSecurityConfig.java (hunk 2 above)
Comment (rodoviti) [reply web/src/main/java/it/pagopa/selfcare/dashboard/web/config/DashboardSecurityConfig.java:30]: we will move this logic into the joint project after [SELC-180](https://pagopa.atlassian.net/browse/SELC-180)

**Review by MarveTedah — APPROVED** [review]

**Review by antonioT90 — APPROVED** [review]

**Pull request merged**